rpcd/playbooks/roles/rpc_setup_infrastructure/tasks/sshd.yml

```diff
@@ -0,0 +1,79 @@
+---
```

Comment:
They could be.  Initially I was going in that direction but decided against it because I think it likely that support may want something like this in the future (to be able to install before osa is laid down).  I didn't have anything specific in mind for what else the role could do, just setting up the framework for future use.

As far as the maxsessions adjustment goes, ya, that can be done via overrides already.  i.e.  +    `line: 'MaxSessions {{ sshd_maxsessions | default(100) }}' `